parachain/pallets/ethereum-beacon-client/Cargo.toml

```diff
@@ -43,11 +43,11 @@ serde_json = "1.0.96"
 hex-literal = { version = "0.4.1" }
 
 [features]
+default = ["std", "fuzzing"]
```

Comment:
Is that required add to `default` feature?  We may not want to build fuzzing tests into production release.

Reply:
I think all our production releases set `default-features = false` for the beacon client. But good point, I'll see if I can remove this anyway. It might be left over from trying to add the `Arbitrary` trait in the beacon client directly, which didn't work anyway.